File: 2273-find-resultant-array-after-removing-anagrams/2273-find-resultant-array-after-removing-anagrams.java
class Solution {
    public List<String> removeAnagrams(String[] words) {
        List<String>list=new ArrayList<>();
        String str="";
        for(int i=0; i<words.length; i++){
            char arr[]=words[i].toCharArray();
            Arrays.sort(arr);
            String s=new String(arr);
            if(!s.equals(str)){
                str=s;
                list.add(words[i]);
            }
        }
        return list;
    }
}

File: 2273-find-resultant-array-after-removing-anagrams/2273-find-resultant-array-after-removing-anagrams.py
class Solution:
    def removeAnagrams(self, words: List[str]) -> List[str]:
        res = []
        prev = ""
        
        for word in words:
            s = ''.join(sorted(word))
            
            if s != prev:
                prev = s
                res.append(word)
                
        return res